_halyard_release_notes/armory-halyard_v1.7.0.md

```diff
@@ -12,10 +12,12 @@ order: -201909131114156
 
 ## Highlights
 
-This release adds support for configuring plugins for Orca as well as a large number of other fixes and improvements. 
+This release adds support for configuring plugins for Orca as well as a large number of other fixes and improvements.
 
 ## Known Issues
-- This release has a bug which prevents `kubeconfig` files from being read from relative paths.  Release `1.7.1` will address this issue and should be released soon.
+This release has a bug which prevents `kubeconfig` files from being read from relative paths.
```

Comment:
You can add something like "workaround: use encrypted secret file reference"

Reply:
I think since 1.7.1 fixes it, it's probably fine to not provide instructions for a workaround